core/chains/evm/config/chain_specific_config.go

```diff
@@ -128,6 +129,7 @@ func setChainSpecificConfigDefaultSets() {
 		gasFeeCapDefault:                      *DefaultGasFeeCap,
 		gasLimitDefault:                       DefaultGasLimit,
 		gasLimitMultiplier:                    1.0,
+		gasLimitMax:                           DefaultGasLimit * 100,
```

Comment:
This isn't significant since only the Arbitrum estimator uses this for now. Perhaps leaving it equal to the default would be more clear.

Reply:
You'd probably also want to verify that applying the multiplier to gasLimitDefault doesn't cause it to exceed this value

Reply:
IIUC the multiplier will not even be used, since no bumping occurs. I agree if we expand this to be a general max though.